libsolidity/codegen/ArrayUtils.cpp

```diff
@@ -1055,28 +1055,27 @@ void ArrayUtils::accessIndex(ArrayType const& _arrayType, bool _doBoundsCheck, b
 	switch (location)
 	{
 	case DataLocation::Memory:
+		// stack: <base_ref> <index>
+		if (!_arrayType.isByteArray())
+			m_context << u256(_arrayType.memoryHeadSize()) << Instruction::MUL;
```

Comment:
(We may want to change this to `_arrayType.baseType()->memoryHeadSize()`, as discussed at https://github.com/ethereum/solidity/pull/6881#discussion_r289808363)

Reply:
Yes, it should be ` _arrayType.baseType()->memoryHeadSize()`